Modelica/Mechanics/Translational.mo

```diff
@@ -1861,14 +1861,15 @@ force law in a target system between two masses.
         end for;
         annotation (Documentation(info="<html>
 <p>
-Returns a table with the friction characteristic table[nTable,2] = [0, f1; ....; v_max, fn], where the first
-column is the velocity v in the range 0..v_max and the second column is the friction force
-according to the Stribeck curve:
+Returns a table with the friction characteristic
+table[nTable,&nbsp;2]&nbsp;=&nbsp;[0,&nbsp;f1;&nbsp;&hellip;;&nbsp;v_max,&nbsp;fn],
+where the first column is the velocity&nbsp;v in the range 0&nbsp;&hellip;&nbsp;v_max
+and the second column is the friction force according to the Stribeck curve:
 </p>
-<pre>
-  F_Coulomb + F_prop*v + F_Stribeck*exp(-fexp*v);
-</pre>
 
+<blockquote><pre>
+f = F_Coulomb + F_prop*v + F_Stribeck*exp(-fexp*v);
+</pre></blockquote>
 </html>"));
```

Comment:
pre needed here?

Reply:
Example codes that refer to existing parameter/variable names should in my opinion always set in `<code>...</code>` or `<pre>`.